network/src/protocols/health_checker/mod.rs

```diff
@@ -244,7 +252,13 @@ where
                                     self.ping_timeout.clone()));
                         }
                         None => {
-                            debug!("{} No connected peer to ping round: {}", self.network_context, self.round);
+                            debug!(
+                                NetworkSchema::new(&self.network_context),
+                                round = self.round,
+                                "{} No connected peer to ping round: {}",
```

Comment:
I wasn't really worried because it's a `debug` level line.  It was more of, let me get all of the logs for a specific network instance, and put them all together.